carma_nav2_port_drayage_demo/src/port_drayage_demo.cpp

```diff
@@ -86,14 +134,66 @@ auto PortDrayageDemo::on_mobility_operation_received(
     pose.header.frame_id = "map";
     pose.pose.position.x = longitude;
```

Comment:
More of a note then a request but still not sure what is expected of how the system will be handling incoming positional data. I think I remember that these mobility operation messages allow for any length of lat/long data but the SAE lengths will cause issues at our scale due to lack of precision.

Reply:
I think there's a coordinate frame discussion to be had here. I've just been passing in map coordinates directly (I think in meters), do we want to pass in GPS coords instead? In theory someone could operate this outside with actual GPS coords, but most of the time it should be fine with map frame.

Reply:
That should be the correct approach for C1T but we may not be able to use this in future work. Probably nothing to worry about now but I assume that we will need to work with actual GPS lat/long values for broader educational purposes.